lib/src/database/hive_collections_database.dart

```diff
@@ -242,41 +242,45 @@ class HiveCollectionsDatabase extends DatabaseApi {
   }
 
   @override
-  Future<void> clear() => transaction(() async {
-        await _clientBox.clear();
-        await _accountDataBox.clear();
-        await _roomsBox.clear();
-        await _roomStateBox.clear();
-        await _roomMembersBox.clear();
-        await _toDeviceQueueBox.clear();
-        await _roomAccountDataBox.clear();
-        await _inboundGroupSessionsBox.clear();
-        await _outboundGroupSessionsBox.clear();
-        await _olmSessionsBox.clear();
-        await _userDeviceKeysBox.clear();
-        await _userDeviceKeysOutdatedBox.clear();
-        await _userCrossSigningKeysBox.clear();
-        await _ssssCacheBox.clear();
-        await _presencesBox.clear();
-        await _timelineFragmentsBox.clear();
-        await _eventsBox.clear();
-        await _seenDeviceIdsBox.clear();
-        await _seenDeviceKeysBox.clear();
-        await _collection.deleteFromDisk();
-      });
+  Future<void> clear({
```

Comment:
prefer making anotherMethod for web clearDBWeb() instead of modify the old one

Reply:
No need

Reply:
Hmm

Reply:
WHAT?

Reply:
why you remove `transaction` @nqhhdev